tests/low-disk/create_and_search_items.py

```diff
@@ -67,22 +95,78 @@ def search_point(qdrant_host, collection_name):
     return resp.json()
 
 
-def initialize_qdrant(qdrant_host, collection_name, points_amount):
-    create_collection(qdrant_host, collection_name)
+def insert_points_and_search(qdrant_host, collection_name, points_amount):
+    collection_json = {
+        "vectors": {
+            "size": VECTOR_SIZE,
+            "distance": "Cosine"
+        }
+    }
+    create_collection(qdrant_host, collection_name, collection_json)
     for points_batch in generate_points(points_amount):
         insert_points(qdrant_host, collection_name, points_batch)
         search_point(qdrant_host, collection_name)
 
 
+def insert_points_then_index(qdrant_host, collection_name, points_amount):
+    """
+    Disable indexing, create collection, insert points up to
+    a point when there is no space left on disk, then start indexing.
+    Wait for indexing then send a search request.
+    @param qdrant_host:
+    @param collection_name:
+    @param points_amount:
+    @return:
+    """
+    collection_json = {
+        "vectors": {
+            "size": VECTOR_SIZE,
+            "distance": "Cosine"
+        },
+        "optimizers_config": {
+            "indexing_threshold": 0,
+            "default_segment_number": 2
+        },
+        "wal_config": {
+            "wal_capacity_mb": 1
+        }
+    }
+    create_collection(qdrant_host, collection_name, collection_json)
+    for points_batch in generate_points(points_amount):
+        res = insert_points(qdrant_host, collection_name, points_batch, quit_on_ood=True)
+        if res == "ood":
+            break
+        search_point(qdrant_host, collection_name)
```

Comment:
I guess a similar comment as above, insert first then search?

Reply:
see the reply above